cmd/snappy/cmd_install.go

```diff
@@ -76,7 +79,12 @@ func (x *cmdInstall) doInstall() error {
 	// TRANSLATORS: the %s is a pkgname
 	fmt.Printf(i18n.G("Installing %s\n"), pkgName)
 
-	realPkgName, err := snappy.Install(pkgName, flags, progress.MakeProgressBar())
+	channel := release.Get().Channel
+	if idx := strings.IndexByte(pkgName, '/'); !helpers.FileExists(pkgName) && idx > -1 {
```

Comment:
I'm preserving the existing behaviour in the `snappy` tool, and am implementing the new behaviour in the `snap` tool.